### PR TITLE
Added blog post about the upcoming Contributor Summit Shanghai.

### PR DESCRIPTION
**Pull request by jberkus**
This blog post is advertisting the contributor summit on June 24th for KubeCon Shanghai.  As such, it would be great to publish it sometime early next week.

Already reviewed by: @idealhack @chenrui333 @puja108 

Docs team, please comment if you can improve the Doc Sprints section: @zacharysarah @jimangel @Bradamant3 @cody-clark

### CODE REVIEW

**Comment by jberkus** [comment]
@chenrui333 my email is in there.  I don't really have another email to use ...

**Comment by chenrui333** [comment]
do we have anything like `community@kubernetes.io` or `kubecon-china-2019@kubernetes.io`?

**Comment by chenrui333** [comment]
Just curious.

**Comment by jberkus** [comment]
We don't.  I can ask about email aliases.

**Comment by chenrui333** [comment]
Sounds good!

**Comment by jberkus** [comment]
@chenrui333 OK to NOT hold the blog post on that, though?  It might take a while.

**Comment by chenrui333** [comment]
yeah, we can totally iterate on this.

File: content/en/blog/_posts/2019-06-11-contributor-summit-shanghai.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+> "Got to know the process from signing CLA to PR and made friends with other contributors." said another.
+
+### Documentation Sprints
+
+Both old and new contributors on our Docs Team will spend a day both improving our documentation and translating it into other languages. If you are interested in having better documentation, fully internationalized into Chinese and other languages, please [sign up](https://www.lfasiallc.com/events/contributors-summit-china-2019/register/) to help with the Doc Sprints.
+
+### Before you attend
+
+Regardless of where you participate, everyone at the Contributor Summit should [sign the Kubernetes CLA](https://git.k8s.io/community/CLA.md#the-contributor-license-agreement) before coming to the conference. You should also bring a laptop suitable for working on documentation or code development.
```
Comment (chenrui333) [inline content/en/blog/_posts/2019-06-11-contributor-summit-shanghai.md]:
```suggestion
Regardless of which you participate, everyone at the Contributor Summit should [sign the Kubernetes CLA](https://git.k8s.io/community/CLA.md#the-contributor-license-agreement) before coming to the conference. You should also bring a laptop suitable for working on documentation or code development.
```

File: content/en/blog/_posts/2019-06-11-contributor-summit-shanghai.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+For the second year, we will have [a Contributor Summit event](https://www.lfasiallc.com/events/contributors-summit-china-2019/) the day before [KubeCon China](https://events.linuxfoundation.cn/events/kubecon-cloudnativecon-china-2019/) in Shanghai. If you already contribute to Kubernetes or would like to contribute, please consider attending and [register](https://www.lfasiallc.com/events/contributors-summit-china-2019/register/). The Summit will be held June 24th, at the Shanghai Expo Center (the same location where KubeCon will take place), and will include a Current Contributor Day as well as the New Contributor Workshop and the Documentation Sprints.
+
+### Current Contributor Day
+
+After last year's Contributor Day, our team received feedback that many of our contributors in Asia and Oceania would like content for current contributors as well. As such, we have added a Current Contributor track to the schedule.
+
+While we do not yet have a full schedule up, the topics covered in the current contributor track will include:
+
+* How to write a KEP
```
Comment (chenrui333) [inline content/en/blog/_posts/2019-06-11-contributor-summit-shanghai.md]:
```suggestion
* How to write a KEP (Kubernetes Enhancement Proposal)
```

File: content/en/blog/_posts/2019-06-11-contributor-summit-shanghai.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+* Codebase and repository review
+* Local Build & Test troubleshooting session
+* Guide to Non-Code Contribution opportunities
+* SIG-Azure face-to-face
```
Comment (chenrui333) [inline content/en/blog/_posts/2019-06-11-contributor-summit-shanghai.md]:
```suggestion
* SIG-Azure face-to-face meeting
```

File: content/en/blog/_posts/2019-06-11-contributor-summit-shanghai.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+* SIG-Scheduling face-to-face
```
Comment (chenrui333) [inline content/en/blog/_posts/2019-06-11-contributor-summit-shanghai.md]:
```suggestion
* SIG-Scheduling face-to-face meeting
```

File: content/en/blog/_posts/2019-06-11-contributor-summit-shanghai.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+The schedule will be on [the Community page](https://github.com/kubernetes/community/tree/master/events/2019/06-contributor-summit) once it is complete.
+
+If your SIG wants to have a face-to-face meeting at Kubecon Shanghai, please contact [Josh Berkus](mailto:jberkus@redhat.com).
+
+### New Contributor Workshop
+
+Students at [last year's New Contributor Workshop](/blog/2018/12/05/new-contributor-workshop-shanghai/) (NCW) found it to be extremely valuable, and the event helped to orient a few of the many Asian and Pacific developers looking to participate in the Kubernetes community.
+
+> "It's a one-stop-shop for becoming familiar with the community," said one participant.
```
Comment (chenrui333) [inline content/en/blog/_posts/2019-06-11-contributor-summit-shanghai.md]:
```suggestion
> "It's a one-stop-shop for becoming familiar with the community." said one participant.
```

File: content/en/blog/_posts/2019-06-11-contributor-summit-shanghai.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+If you have not contributed to Kubernetes before, or have only done one or two things, please consider [enrolling](https://www.lfasiallc.com/events/contributors-summit-china-2019/register/) in the NCW.
+
+> "Got to know the process from signing CLA to PR and made friends with other contributors." said another.
+
+### Documentation Sprints
+
+Both old and new contributors on our Docs Team will spend a day both improving our documentation and translating it into other languages. If you are interested in having better documentation, fully internationalized into Chinese and other languages, please [sign up](https://www.lfasiallc.com/events/contributors-summit-china-2019/register/) to help with the Doc Sprints.
```
Comment (chenrui333) [inline content/en/blog/_posts/2019-06-11-contributor-summit-shanghai.md]:
```suggestion
Both old and new contributors on our Docs Team will spend a day both improving our documentation and translating it into other languages. If you are interested in having better documentation, fully localized into Chinese and other languages, please [sign up](https://www.lfasiallc.com/events/contributors-summit-china-2019/register/) to help with the Doc Sprints.
```

File: content/en/blog/_posts/2019-06-11-contributor-summit-shanghai.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+**Author**: Josh Berkus (Red Hat)
+
+![Picture of contributor panel at 2018 Shanghai contributor summit.  Photo by Josh Berkus, licensed CC-BY 4.0](/images/blog/2019-
+06-11-contributor-summit-shanghai/panel.png)
+
+For the second year, we will have [a Contributor Summit event](https://www.lfasiallc.com/events/contributors-summit-china-2019/) the day before [KubeCon China](https://events.linuxfoundation.cn/events/kubecon-cloudnativecon-china-2019/) in Shanghai. If you already contribute to Kubernetes or would like to contribute, please consider attending and [register](https://www.lfasiallc.com/events/contributors-summit-china-2019/register/). The Summit will be held June 24th, at the Shanghai Expo Center (the same location where KubeCon will take place), and will include a Current Contributor Day as well as the New Contributor Workshop and the Documentation Sprints.
+
+### Current Contributor Day
+
+After last year's Contributor Day, our team received feedback that many of our contributors in Asia and Oceania would like content for current contributors as well. As such, we have added a Current Contributor track to the schedule.
```
Comment (chenrui333) [inline content/en/blog/_posts/2019-06-11-contributor-summit-shanghai.md]:
```suggestion
After last year's Contributor Day, our team received some feedback that many of our contributors in Asia and Oceania would like contents for current contributors as well. 
As such, we have added a Current Contributor track to the schedule.
```

File: content/en/blog/_posts/2019-06-11-contributor-summit-shanghai.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+title: 'Join us at the Contributor Summit in Shanghai'
+date: 2019-06-11
+---
+
+**Author**: Josh Berkus (Red Hat)
+
+![Picture of contributor panel at 2018 Shanghai contributor summit.  Photo by Josh Berkus, licensed CC-BY 4.0](/images/blog/2019-
+06-11-contributor-summit-shanghai/panel.png)
+
+For the second year, we will have [a Contributor Summit event](https://www.lfasiallc.com/events/contributors-summit-china-2019/) the day before [KubeCon China](https://events.linuxfoundation.cn/events/kubecon-cloudnativecon-china-2019/) in Shanghai. If you already contribute to Kubernetes or would like to contribute, please consider attending and [register](https://www.lfasiallc.com/events/contributors-summit-china-2019/register/). The Summit will be held June 24th, at the Shanghai Expo Center (the same location where KubeCon will take place), and will include a Current Contributor Day as well as the New Contributor Workshop and the Documentation Sprints.
```
Comment (chenrui333) [inline content/en/blog/_posts/2019-06-11-contributor-summit-shanghai.md]:
```suggestion
For the second year, we will have [a Contributor Summit event](https://www.lfasiallc.com/events/contributors-summit-china-2019/) the day before [KubeCon China](https://events.linuxfoundation.cn/events/kubecon-cloudnativecon-china-2019/) in Shanghai. If you have already contributed to Kubernetes or would like to contribute, please consider attending and [register](https://www.lfasiallc.com/events/contributors-summit-china-2019/register/). The Summit will be held on June 24th, at the Shanghai Expo Center (the same exact location where KubeCon will take place), and will include a Current Contributor Day, the New Contributor Workshop, and the Documentation Sprints.
```

**Comment by chenrui333** [comment]
@jberkus During the translation, I find some potential improvements that we can make on the article. Please check them out.

File: content/en/blog/_posts/2019-06-11-contributor-summit-shanghai.md (hunk 1 above)
Comment (sftim) [reply content/en/blog/_posts/2019-06-11-contributor-summit-shanghai.md]: My suggestion:
```suggestion
Regardless of where you participate, everyone at the Contributor Summit should [sign the Kubernetes Contributor License Agreement](https://git.k8s.io/community/CLA.md#the-contributor-license-agreement) (CLA) before coming to the conference. You should also bring a laptop suitable for working on documentation or code development.
```

File: content/en/blog/_posts/2019-06-11-contributor-summit-shanghai.md (hunk 7 above)
Comment (jberkus) [reply content/en/blog/_posts/2019-06-11-contributor-summit-shanghai.md]: Doesn't work in idiomatic English, I'm afraid.

File: content/en/blog/_posts/2019-06-11-contributor-summit-shanghai.md (hunk 8 above)
Comment (jberkus) [reply content/en/blog/_posts/2019-06-11-contributor-summit-shanghai.md]: Same as above.  The original sentence is more "natural".

File: content/en/blog/_posts/2019-06-11-contributor-summit-shanghai.md (hunk 7 above)
Comment (chenrui333) [reply content/en/blog/_posts/2019-06-11-contributor-summit-shanghai.md]: ok. 😃

**Pull request merged**